probe-rs/src/config/registry.rs

```diff
@@ -45,6 +48,7 @@ impl std::fmt::Display for RegistryError {
 
         match self {
             ChipNotFound => write!(f, "The requested chip was not found."),
+            ChipAutodetectFailed => write!(f, "The detected chip is unknown."),
```

Comment:
I think the error message should be more general. I think it could also happen that the auto-detect fails for a different reason, e.g. some communication problem.